com.gopeed.Gopeed.yaml

```diff
@@ -0,0 +1,54 @@
+app-id: com.gopeed.Gopeed
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+rename-icon: gopeed
+rename-desktop-file: gopeed.desktop
+command: gopeed
+finish-args:
+  - '--share=ipc'
+  - '--socket=x11'
+  - '--share=network'
+  - '--device=dri'
+  - '--talk-name=org.freedesktop.Notifications'
+  # https://github.com/miguelpruivo/flutter_file_picker/pull/1275
+  - '--filesystem=xdg-download'
+  - '--filesystem=/tmp'
```

Comment:
Why is `/tmp` back?

Reply:
I'm sorry, I thought Flutter didn't support TMPDIR env until I found https://github.com/flutter/plugins/pull/4218